be/src/runtime/load_channel.cpp

```diff
@@ -95,6 +95,7 @@ void LoadChannel::add_chunk(brpc::Controller* cntl, const PTabletWriterAddChunkR
 void LoadChannel::cancel() {
     _span->AddEvent("cancel");
     auto scoped = trace::Scope(_span);
```

Comment:
It's better to print the tablet_id list?

Reply:
Some table may have 10,000 tablets